Xamarin.Essentials/Preferences/Preferences.win32.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Preferences
+    {
+        static readonly string settingsPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), AppInfo.Name, "settings.dat");
+
+        static readonly Dictionary<string, Dictionary<string, object>> preferences = new Dictionary<string, Dictionary<string, object>>();
```

Comment:
This is a tad bit nit-picky, but it might be nicer to just add a usings to the top of the file:

```csharp
using PreferencesDictionary = System.Collections.Generic.Dictionary<string, System.Collections.Generic.Dictionary<string, object>>;
```
This way, we don't have angle brackets all over the show.